src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -68,16 +69,41 @@ public class PreferencesViewModel : ViewModelBase, INotifyPropertyChanged
         private bool isVisibleAddStyleBorder;
         private bool isEnabledAddStyleButton;
         private GeometryScalingOptions optionsGeometryScale = null;
-
+        private GeometryScaleSize defaultGeometryScaling = GeometryScaleSize.Medium;
+        private GeometryScaleSize currentGeometryScaling;
         #endregion Private Properties
 
-        public GeometryScaleSize ScaleSize { get; set; }
+        public GeometryScaleSize CurrentGeometryScaling
```

Comment:
do we want to move this property out? It is no longer related to preferences

Reply:
This property was deleted since it was duplicating the functionality that the one in GeometryScalingViewModel, please check the next commit: https://github.com/DynamoDS/Dynamo/pull/13656/commits/789f51c4a3c106156940924585e1ca40abd9842b